containers/Ideas/IdeaCreate.js

```diff
@@ -6,6 +6,7 @@ import * as yup from 'yup';
 import AsyncStorage from '@react-native-async-storage/async-storage';
 import Icon from 'react-native-vector-icons/SimpleLineIcons';
 import IconSLI from 'react-native-vector-icons/SimpleLineIcons';
+import DropDownPicker from 'react-native-dropdown-picker';
```

Comment:
I guess github actions is right here: should be removed if it's not used.

Reply:
Its removed!